azext_iot/digitaltwins/providers/model.py

```diff
@@ -7,7 +7,8 @@
 import json
 from azext_iot.common.utility import process_json_arg, scantree, unpack_msrest_error
 from azext_iot.digitaltwins.providers.base import DigitalTwinsProvider
-from azext_iot.digitaltwins.providers import ErrorResponseException
+# from azext_iot.digitaltwins.providers import ErrorResponseException
+from azext_iot.sdk.digitaltwins.dataplane.models.error_response_py3 import ErrorResponseException
```

Comment:
you should be able to import right from models i.e. `azext_iot.sdk.digitaltwins.dataplane.models` as `ErrorResponseException` is imported in `init`.